containers/iam-rucio-sync/Dockerfile

```diff
@@ -1,50 +1,13 @@
-ARG BASEIMAGE=rucio/rucio-server
-ARG BASETAG=release-1.30.0
-# ARG RUCIO_VERSION=1.30.0
-
-FROM $BASEIMAGE:$BASETAG
-
-# cleanup yum cache
-RUN yum upgrade -y && \
-yum clean all && \
-rm -rf /var/cache/yum
-
-# install useful tools
-RUN yum -y install git htop voms-clients-cpp
+ARG BASETAG=latest
+ 
+# To be changed to specific version+hsa
+FROM ghcr.io/vre-hub/vre-base-layer-ops:${BASETAG}
+LABEL maintainer="VRE Team @ CERN"
 
 WORKDIR /home/user/
 
-# # clone repos
+# # clone repo
 RUN git clone https://github.com/vre-hub/vre.git
```

Comment:
Maybe this should also be versioned, as the files will change - what do you think?